public/static/docs/use-cases/shared-development-server.md

```diff
@@ -46,6 +56,13 @@ running:
 $ dvc config cache.dir /path/to/dvc-cache
 ```
 
+And tell DVC to set group permissions on the newly created/downloaded cache
+files:
+
+```dvc
+$ dvc config cache.shared group
+```
+
```

Comment:
The reason why it wasn't needed before is that I've assumed that people will just create their files with proper group permissions and so our cache files won't need any changes. That was clearly too much to ask, so we now do it explicitly.